tools/meta-bake/meta-bake.py

```diff
@@ -298,6 +300,12 @@ def vab(self, cfg: dict, repos: dict, ctx: dict, args: argparse.Namespace, targe
         # copy uboot src to folder uboot-socfpga-vab-n6000
         shutil.copytree(uboot_src_dir, uboot_vab)
 
+        for i in range(len(linux_binary)):
+            print(linux_binary[i])
```

Comment:
I think the more pythonistic way of the loop would be

for fname in linux_binary:
    print(fname)